sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -135,13 +110,17 @@ public:
   bfloat16(const float &a);
```

Comment:
This comment is for these lines (above):

```
// Explicit conversion functions
static storage_t from_float(const float &a);
static float to_float(const storage_t &a);
```

I see that these have been removed from the `public` section of the class in the header.  They should therefore be removed from this specification.

Reply:
Yes, done.